awx/main/expect/run.py

```diff
@@ -14,10 +14,7 @@
 import sys
 import threading
 import time
-try:
```

Comment:
I added this weeks ago to facilitate isolated nodes still using py2.  We don't need it anymore since none of this code runs on isolated nodes anymore (it's all py3 + runner).